web_src/css/repo.css

```diff
@@ -1066,6 +1066,9 @@
 .repository.view.issue .comment-list .event .detail {
   margin-top: 4px;
   margin-left: 14px;
+  display: flex;
+  align-items: center;
+  gap: 0.25rem;
```

Comment:
I think it could simply use `flex-text-block` in HTML

![image](https://github.com/go-gitea/gitea/assets/2114189/36df1fa5-2354-4b99-8c2b-c0bb494d7265)